### PR TITLE
Update ARIA descriptions and other docs tweaks

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Update ARIA property descriptions for consistency and to address https://github.com/blackbaud/skyux2-docs/issues/919, plus clean up a few other minor docs things I noticed while checking the component docs in this repo.

### CODE REVIEW

File: src/app/public/modules/checkbox/checkbox.component.ts [hunk 1]
```diff
@@ -39,21 +39,21 @@ let nextId = 0;
...
-   * [accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
-   * The value should be the HTML element ID (without the leading `#`) of the element
-   * that labels the checkbox. If the checkbox does not include a visible label on the screen, use the `label` property instead.
+   * Specifies the HTML element ID (without the leading `#`) of the element that labels the
+   * checkbox. This sets the checkbox's `aria-labelledby` attribute
+   * [to support accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
+   * If the checkbox does not include a visible label, use `label` instead.
    */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/checkbox/checkbox.component.ts:56]:
```suggestion
  @Input()
```

File: src/app/public/modules/checkbox/checkbox.component.ts [hunk 2]
```diff
@@ -39,21 +39,21 @@ let nextId = 0;
...
-   * [accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility)
-   * when the checkbox does not include a visible label. This property must be set if
-   * you are using an icon checkbox. If the checkbox includes a visible label on the screen, use the `labelledBy` property instead.
+   * Specifies an ARIA label for the checkbox. This sets the checkbox's `aria-label` attribute
+   * [to support accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility)
+   * when the checkbox does not include a visible label. You must set this property for icon
+   * checkboxes. If the checkbox includes a visible label, use `labelledBy` instead.
    */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/checkbox/checkbox.component.ts:47]:
```suggestion
  @Input()
```

File: src/app/public/modules/radio/radio-group.component.ts [hunk 3]
```diff
@@ -50,19 +50,21 @@ let nextUniqueId = 0;
...
- * Sets the radio button group's `aria-labelledby` attribute to support accessibility. The value
- * should be the HTML element ID (without the leading `#`) of the element that labels the radio
- * button group. If the radio button group does not include a visible label, use the `ariaLabel` property instead.
+ * Specifies the HTML element ID (without the leading `#`) of the element that labels
+ * the radio button group. This sets the radio button group's `aria-labelledby` attribute
+ * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+ * If the radio button group does not include a visible label, use `ariaLabel` instead.
  */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/radio/radio-group.component.ts:58]:
```suggestion
  @Input()
```

File: src/app/public/modules/radio/radio-group.component.ts [hunk 4]
```diff
@@ -50,19 +50,21 @@ let nextUniqueId = 0;
...
- * Defines a string value to label the radio button group and set the `aria-label`
- * attribute to support accessibility. If the radio button group includes a visible
- * label, use the `ariaLabelledBy` property instead.
+ * Specifies an ARIA label for the radio button group. This sets the
+ * radio button group's `aria-label` attribute
+ * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+ * If the radio button group includes a visible label, use `ariaLabelledBy` instead.
  */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/radio/radio-group.component.ts:67]:
```suggestion
  @Input()
```

File: src/app/public/modules/radio/radio.component.ts [hunk 5]
```diff
@@ -109,21 +109,21 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
...
- * radio button does not include a visible label. You must set this property if you
- * are using an icon radio button. If the radio button includes a visible label,
- * use the `labelledBy` property instead.
+ * Specifies an ARIA label for the radio button. This sets the radio button's `aria-label`
+ * attribute [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility)
+ * when the radio button does not include a visible label. You must set this property for icon
+ * radio buttons. If the radio button includes a visible label, use `labelledBy` instead.
  */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/radio/radio.component.ts:117]:
```suggestion
  @Input()
```

File: src/app/public/modules/radio/radio.component.ts [hunk 6]
```diff
@@ -109,21 +109,21 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
...
- * accessibility. The value should be the HTML element ID (without the leading `#`) of the
- * element that labels the radio button. If the radio button does not include a visible label,
- * use the `label` property instead.
+ * Specifies the HTML element ID (without the leading `#`) of the element that labels
+ * the radio button. This sets the radio button's `aria-labelledby` attribute
+ * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+ * If the radio button does not include a visible label, use `label` instead.
  */
-  @Input()
+@Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/radio/radio.component.ts:126]:
```suggestion
  @Input()
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-aria-link-120320](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-aria-link-120320)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-aria-link-120320](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-aria-link-120320)

**Pull request merged**